specification/columns/chargeperiodstart.md

```diff
@@ -2,7 +2,7 @@
 
 Charge Period Start represents the [*inclusive*](#glossary:inclusivebound) start date and time within a [*charge period*](#glossary:chargeperiod). For example, a time period where ChargePeriodStart is '2024-01-01T00:00:00Z' and [ChargePeriodEnd](#chargeperiodend) is '2024-01-02T00:00:00Z' includes charges for January 1, since ChargePeriodStart is *inclusive*, but does not include charges for January 2 since ChargePeriodEnd is [*exclusive*](#glossary:exclusivebound).
 
-ChargePeriodStart MUST be present in the billing data, MUST be of type Date/Time, MUST be an *inclusive* value, and MUST NOT contain null values.
+ChargePeriodStart MUST be present in the billing data, MUST be of type Date/Time, MUST be an *inclusive* value, and MUST NOT contain null values. ChargePeriodStart MUST match the beginning date and time boundary for which a charge is active.
```

Comment:
@aqu-erp Do you think the glossary needs updating as well? There we describe it as when the charge was incurred (not active or applied).

Reply:
I think "incurred" sounds more accurate here.

> ChargePeriodStart MUST match the beginning date and time boundary for when a charge was incurred.

Reply:
The concern that was raised in #441 was about how can we describe scenarios where a purchase is made for the future. E.g. an RI was purchased on May 8th that should apply the discount from May 14th, 2024 - May 13, 2025. Is the Charge incurred on May 8th? Or May 14th? I think we would like to see the period start and end be the 14th of May, 2024 - May 13, 2025. If Incurred explains that, I'm good with it. If we need different wording, open to that as well. Thoughts?